### PR TITLE
SCRD-4750 Add logout button to Day 2 UI

### PR DESCRIPTION
**Pull request by jeremy-moffitt**
SCRD-4750 Added the logout button to the masthead

### CODE REVIEW

**Comment by jgravel** [comment]
I agreed with Gary and I wished this change had been there when I worked on the login page. The change worked but I wonder if we should have a tool tip or some explanation for the logout button. The icon is self-explanatory for some people but having some text description would clear all the doubts. This is not a big issue. It can be done in a separate pull request. By the way, the SCC website uses the 'Logout' word instead of an icon.

**Comment by jeremy-moffitt** [comment]
added a tooltip for the button, kept it as an icon for now... will include that on the items we ask for UX review in the near future

File: src/components/NavMenu.js [hunk 1]
```diff
@@ -76,6 +79,13 @@ class NavMenu extends Component {
       });
     });
 
+    let logout = () => {
+      clearAuthToken();
+      redirectToLogin(false);
+    }
+
+    const logout_tooltip = (<Tooltip id='logout' className='tooltip'>{translate('logout')}</Tooltip>);
```
Comment (jgravel) [inline src/components/NavMenu.js]: I don't see the 'tooltip' class defined anywhere. Do we need it?

Comment (jeremy-moffitt) [reply src/components/NavMenu.js]: I copied that from some other code, I'll try without it

**Pull request merged**